ci-operator/step-registry/openshift/selfupgrade/aws/loki/OWNERS

```diff
@@ -0,0 +1,12 @@
+approvers:
```

Comment:
Unless there's a good reason to manage this step registry's permissions separately, consider symlinking the ci-operator-step-registry/openshift/OWNERS (as is the precedent for most of the other e2e workflows).

Reply:
I suppose, although symlinks for perms are tricky. Do you have any idea why this wasn't setup with OWNER_ALIASES?

Reply:
I think that's a question for dptp, I'm just striving for consistency. If/when a refactor is required the job will be easier if workflows follow the same pattern consistently.